api/internal/handler/route/route.go

```diff
@@ -307,6 +307,12 @@ func generateLuaCode(script map[string]interface{}) (string, error) {
 
 func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 	input := c.Input().(*entity.Route)
+	// check duplicate name
+	if err := h.checkDuplicateName(c, input.Name, ""); err != nil {
```

Comment:
should add this check for `Update` too. Thanks.

Reply:
ok